js/src/util/template-factory.js

```diff
@@ -143,7 +143,7 @@ class TemplateFactory extends Config {
   }
 
   _resolvePossibleFunction(arg) {
-    return execute(arg, [this])
+    return execute(arg, [undefined, this])
```

Comment:
```suggestion
    return typeof arg === 'function' ? arg.call(this._element) : arg
```

I suppose it is better to change only this, as it is count as a regression, and add a test, instead of messing all the rest calls

Reply:
`TemplateFactory` doesn't have access to the underlying element, AFAICT. There are two calls where the `this` argument actually matters:

- `_resolvePossibleFunction` in `js/src/tooltip.js`
- The call to `execute(this._config.placement, ...)` in `_createPopper` in `js/src/tooltip.js`

I can update those two call sites to use the `typeof arg === 'function' ...` pattern if you'd prefer that I avoid changing `execute`?